tests/PHPUnit/Unit/Scheduler/TimetableTest.php

```diff
@@ -61,6 +63,19 @@ public function testGetTimetableFromOptionValue($expectedTimetable, $option)
         self::resetPiwikOption();
     }
 
+    public function testRescheduleTaskAndRunTomorrow()
+    {
+        $timetable = new Timetable();
```

Comment:
This test [fails](https://travis-ci.org/piwik/piwik/jobs/227048006#L857), as no database is set up, but `Timetable` constructor performs an `Option:get`, which requires a database.